components/service/fretboard/src/main/java/mozilla/components/service/fretboard/Fretboard.kt

```diff
@@ -45,17 +45,19 @@ class Fretboard(
      * saves them to local storage
      */
     @Synchronized
-    fun updateExperiments() {
+    fun updateExperiments(): Boolean {
         if (!experimentsLoaded) {
             loadExperiments()
         }
         try {
             val serverExperiments = source.getExperiments(experimentsResult)
             experimentsResult = serverExperiments
             storage.save(serverExperiments)
+            return true
         } catch (e: ExperimentDownloadException) {
             // Keep using the local experiments
             logger.error(e.message, e)
+            return false
```

Comment:
nit you can move the return outside, AS may even recommend that:

```Kotlin
return try {
...
}
```

Reply:
Yes, I forgot about that!